kafka/README.md

```diff
@@ -0,0 +1,37 @@
+# Kafka SpecMesh extension
+
+Extends SpecMesh with the concept of public, protected and public topics.
+
+## Authorisation
+
+## Topic authorisation
+
+The provisioner will set ACLs to enforce the following topic authorisation:
+
+### Public topics:
+
+Only the domain itself can `WRITE` to public topics; Any domain can `READ` from them.
```

Comment:
public topics are denoted using a relative path from the domain and prefixed with '_public'. The resulting topics will be domain.structure._public.xxx.yyy

Reply:
I've added something, but again, this is outside the scope of what I was doing in this PR. 

We definitely need to write some docs on what we already have... https://github.com/specmesh/specmesh-build/issues/64